web/src/components/ContrastThemeToggle.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { ContrastIcon } from '../assets'
+import { useContrastTheme } from '../hooks/useContrastTheme'
+import useWindowDimensions from '../hooks/useWindowDimensions'
+import ToolbarItem from './ToolbarItem'
+import Button from './base/Button'
+import Icon from './base/Icon'
+
+const ContrastButton = styled(Button)`
+  display: flex;
+  width: 100%;
```

Comment:
🔧 Not needed
```suggestion
```

Reply:
width is needed so that its children could move on language direction change

Reply:
True 👍